src/app/api/routes/media.py

```diff
@@ -49,3 +66,26 @@ async def delete_media(media_id: int = Path(..., gt=0)):
     Based on a media_id, deletes the given media
     """
     return await crud.delete_entry(media, media_id)
+
+
+@router.post("/{media_id}/upload_file", response_model=MediaOut, status_code=200)
+async def upload_media(media_id: int = Path(..., gt=0),
+                       file: UploadFile = File(...),
+                       current_device: DeviceOut = Security(get_current_device, scopes=["device"])):
+    existing_media = await crud.fetch_one(media, [("id", media_id), ("device_id", current_device.id)])
+    if existing_media is None:
+        raise HTTPException(
+            status_code=400,
+            detail="Permission denied"
+        )
+
+    bucket_key = existing_media["bucket_key"]
+    upload_success = await bucket_service.upload_file(bucket_name="mypyroneartest",
```

Comment:
shoud we put the bucketname in the config?

Reply:
good point. Done